packages/xo-web/src/xo-app/xosan/new-xosan.js

```diff
@@ -411,6 +411,7 @@ export default class NewXosan extends Component {
                     columns={XOSAN_SR_COLUMNS}
                     data-hosts={this._getHosts()}
                     onSelect={this._selectSrs}
+                    stateUrlParam='sr'
```

Comment:
I don't think this is good because `'sr'` could be used for something else (e.g. auto-selecting a SR like the `pool` param in new-vm).
So I suggest we prefix all the `SortedTable`'s `stateUrlParam`s with `'s_'` (`'s_srs'`, `'s_logs'`, etc.) and keep using `'s'` when it makes sense for the page to only contain one `SortedTable` as its main content (e.g. the home page, the vm/disks page, etc.).

Reply:
IMHO, we should avoid changing the URL param associated to a table because it would invalidate possible users bookmarks.

Therefore we should favored qualified URL param names in order to avoid changing them when a new table appears on the same page.

Reply:
So you mean to use `'s_vdis'`, for example, even though the VDIs table is the main content of the page?

Reply:
I agree with you, sorry for the noise :upside_down_face: